src/style/app-fixed.module.css

```
@@ -801,13 +801,58 @@
 .modalHeader {
   border: none;
   padding-bottom: 0;
-  background-color: var(--modalHeader-bg) !important;
+  background-color: #f5f5f5 !important;
   padding: 1rem 1rem;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)